modules/ixBidAdapter.js

```diff
@@ -311,6 +313,19 @@ function getEidInfo(allEids) {
       }
     }
   }
+
+  const isValidFlocId = utils.deepAccess(flocData, 'id') && utils.deepAccess(flocData, 'version');
+
+  if (isValidFlocId) {
+    var flocEid = {
```

Comment:
Prefer const/let and be consistent within the codebase

Reply:
ok, thanks